index.html

```diff
@@ -297,23 +298,34 @@ <h3 class="c-hand" data-micromodal-trigger="modal-credits">Le site</h3>
-                            </a>
+                    <div>
+                        <div class="wrapper">
+                            <div class="one">
+                                <a href="https://www.kanoma.fr" target="_blank" rel="noopener noreferrer">
+                                    <img src="img/sponsors/kanoma.png" alt="Kanoma"/>
+                                </a>
+                            </div>
+                            <div class="two">
+                                <a href="https://www.kokan.fr" target="_blank" rel="noopener noreferrer">
+                                    <img src="img/sponsors/kokan.png" alt="Kokan"/>
+                                </a>
+                            </div>
+                            <div class="three">
+                                <a href="https://www.shodo.io" target="_blank" rel="noopener noreferrer">
+                                    <img src="img/sponsors/shodo.jpg" alt="Shodo"/>
```

Comment:
```suggestion
                                    <img src="img/sponsors/shodo.png" alt="Shodo"/>
```
avec la nouvelle image :-)